m4/pbs_default_scheduler.m4

```diff
@@ -37,12 +37,7 @@
 
 AC_DEFUN([PBS_AC_DEFAULT_SCHEDULER],
 [
-  AC_MSG_CHECKING([for Default scheduler name])
```

Comment:
Here is the usecase - 
sched_name is required in qmgr, scheduler, server. Is there any file which can be accessed in all the components? As far as i understand  the answer is pbs_config.h. This file is auto generated from autotools.  

Other wise we need to define a new header file. I feel adding AC macro is better way.